acquia_lift.module

```diff
@@ -2118,21 +2118,30 @@ function acquia_lift_form_personalize_agent_option_sets_form_alter(&$form, &$for
 /**
  * Implements hook_form_FORM_ID_alter().
  *
- * Alters the status change form so that legacy campaigns whose status is completed
- * cannot be changed to any other status.
+ * Alters the status change form so that:
+ *   1) If the personalization is already started, render the buttons;
+ *      otherwise link to the review page.
+ *   2) Legacy campaigns whose status is completed cannot be changed to any other status.
  */
 function acquia_lift_form_personalize_status_change_form_alter(&$form, &$form_state) {
   $agent_name = $form['agent_name']['#value'];
+  $agent = personalize_agent_load($agent_name);
+  $status = personalize_agent_get_status($agent_name);
+  // If the personalization is already started, render the buttons; otherwise link to the review page.
+  if (acquia_lift_is_targeting_agent($agent) && $status === PERSONALIZE_STATUS_NOT_STARTED) {
```

Comment:
Hmm, if we want to make the behavior different, is there some way we can check where the form is being rendered?